docs/reference/predicates.md

```diff
@@ -716,6 +716,32 @@ catalog:
     "https://catalog";
 ```
 
+## TrafficSegment
+
+TrafficSegment predicate requires two number arguments _min_ and _max_
+from an interval [0, 1] (from zero included to one included) and _min_ <= _max_.
+
+Let _r_ be one-per-request uniform random number value from [0, 1).
+TrafficSegment matches if _r_ belongs to an interval from [_min_, _max_).
+Upper interval boundary _max_ is excluded to simplify definition of
+adjacent intervals - the upper boundary of the first interval
+then equals lower boundary of the next and so on, e.g. [0, 0.25) and [0.25, 1).
+
+This predicate has weight of -1 and therefore does not affect route weight.
+
+Parameters:
+
+* min (decimal) from an interval [0, 1]
+* max (decimal) from an interval [0, 1], min <= max
+
+Example of routes splitting traffic in 50%+30%+20% proportion:
```

Comment:
Maybe also tell about the advantage compared to `Traffic`, because the new one does not need to use `True` hacks in order to have the routes ordered correctly in case we have multi color traffic switching.

Reply:
`Traffic` and `TrafficSegment` are low-level plumbing. If we accept `TrafficSegment` next step I plan is to re-implement Ingress and RouteGroup backend weight using it and there we can describe why we change it and its benefits.